lib/vrt/mapping.rb

```diff
@@ -50,14 +56,35 @@ def load_mappings
     # becomes
     #     {one: {'id': 'one', 'foo': 'bar'}, two: {'id': 'two', 'foo': 'baz'}}
     def key_by_id(mapping)
```

Comment:
for this method it would be nice to have a comment for each condition simply stating an example mapping that meets that condition.

Reply:
I gave this a shot but didn't think I could improve on the comment above this method and the readability of the code. If you have a specific idea here feel free to just push a commit with the comments to this branch.

Reply:
no worries, i agree should be self-explanatory